1-exercises/C-while-loop-with-array/exercise.js

```diff
@@ -17,7 +17,16 @@ const BIRTHDAYS = [
 ];
 
 function findFirstJulyBDay(birthdays) {
-    // TODO
+     let i = 0;
+     let birthday;
+     while (i < birthdays.length) {
+        if (birthdays[i].includes("July")) {
+             birthday = birthdays[i];
+             i++
+        }
+        i++;
+     }
+    return birthday
```

Comment:
A well structured implementation of the function. However, the function will continue to search the presence of July up till the end even after it found the first July. The presence of `i++` in line27 made the function not to give wrong output. Because it skips the next July in the array. If you try to input another July birthday after September 28th, you will see that the function will return the last July instead of the first. I suggest you end the while loop immediately you find the occurrence of July. It is also a good practice to put semi-colon after statements such as return in line 29.